### PR TITLE
Update to latest RFEM

### PR DESCRIPTION
**Pull request by OndraMichal**
# Description
This change concludes all changes needed so that client can work with latest RFEM. All versions 6.02.0065 should work.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# How Has This Been Tested?

- [x] Unit Tests
- [ ] Attached examples

**Test Configuration**:
* RFEM / RSTAB version: 6.02.0065.149.7baec1cb242
* Python version: 3.11.3

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: RFEM/TypesForMembers/memberSupport.py [hunk 1]
```diff
@@ -60,23 +56,16 @@ def __init__(self,
         clientObject.spring_translation_x = spring_translation_x
         clientObject.spring_translation_y = spring_translation_y
         clientObject.spring_translation_z = spring_translation_z[0]
-        clientObject.nonlinearity = spring_translation_z[1].name
```
Comment (heetrojivadiya) [inline RFEM/TypesForMembers/memberSupport.py:63]: In newer version, name of nonlinearity for translation_z is changed to 'nonlinearity_translational_z' but it is available and accessible with wsdl.

image

image

File: RFEM/initModel.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 
 # Check if port is listening
 a_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+#a_socket.settimeout(360)
```
Comment (heetrojivadiya) [inline RFEM/initModel.py:26]: commented code line should be removed

**Review by heetrojivadiya — CHANGES_REQUESTED** [review]

**Review by heetrojivadiya — COMMENTED** [review]
changes in new commit seems okay and all new changed unittests are passed. Still two previous comments are not addressed in this commit.

File: RFEM/TypesForMembers/memberShearPanel.py [hunk 3]
```diff
@@ -288,19 +252,13 @@ def DefineSProv(no: int = 1,
         clientObject.definition_type = MemberShearPanelDefinitionType.DEFINITION_TYPE_DEFINE_S_PROV.name
 
         # Member Shear Panel Assigned Member Supports
-        clientObject.member_supports = ConvertToDlString(member_supports)
+        #clientObject.member_supports = ConvertToDlString(member_supports)
```
[the remark this answers is not in the document]
Comment (OndraMichal) [reply RFEM/TypesForMembers/memberShearPanel.py]: fixed

File: RFEM/TypesForMembers/memberSupport.py (hunk 1 above)
Comment (OndraMichal) [reply RFEM/TypesForMembers/memberSupport.py:63]: Fixed

File: RFEM/BasicObjects/member.py [hunk 4]
```diff
@@ -487,7 +487,6 @@ def Rib(
             end_section_no: int = 1,
             rib_surfaces_no =  [],
             rib_alignment = MemberTypeRibAlignment.ALIGNMENT_ON_Z_SIDE_POSITIVE,
-            reference_width_type = MemberReferenceLengthWidthType.REFERENCE_LENGTH_WIDTH_SIXTH,
```
Comment (heetrojivadiya) [inline RFEM/BasicObjects/member.py:490]: It should be also updated in docstrings too.

**Pull request merged**